record.c

```diff
@@ -255,6 +256,33 @@ janus_recorder *janus_recorder_create_full(const char *dir, const char *codec, c
 	return rc;
 }
 
+int janus_recorder_pause(janus_recorder *recorder) {
+	if(!recorder)
+		return -1;
+	janus_mutex_lock_nodebug(&recorder->mutex);
+	g_atomic_int_set(&recorder->paused, TRUE);
+	janus_mutex_unlock_nodebug(&recorder->mutex);
```

Comment:
There's no need for a mutex, here: it's an atomic operation. Besides, the value should be `1` and not `TRUE`.